### PR TITLE
converted create your first snap tutorial

### LINKED ISSUE #269: convert- # Create your first snap tutorial
1hA-ytZZjTxajh8vIIpV6MUE03poBjdJizDQVUBgurhM

### PR DESCRIPTION
**Pull request by evilnick**
converted from gdoc to markdown

Still needs style review!

Fixes #269

### CODE REVIEW

**Comment by evilnick** [comment]
jaas-xxxx-_039

jaas-xxxx-_041

File: tutorials/snapcraft/create-your-first-snap.md [hunk 1]
```diff
@@ -0,0 +1,726 @@
...
+description: |
+  GNU hello prints a friendly greeting.
+grade: devel
+confinement: devmode
+```
+
+
+```bash
+A quick note on the **version** bit: it’s put into quotation marks ('`2.10`') here, because the version is a string, not a number. You could theoretically use a version string without any numbers (like '`myfirstversion`') and that would be fine too. This version is only for user consumption and doesn’t require anything like ascii ordering (e.g ver1 > ver2 and the like) for an update to reach the user.
+```
```
Comment (degville) [inline tutorials/snapcraft/create-your-first-snap.md]: This is an admonition in the original:

image

File: tutorials/snapcraft/create-your-first-snap.md [hunk 2]
```diff
@@ -0,0 +1,726 @@
...
+: Lost or starting from here?
+Check or download [here][here2] to see what your current directory should look like.
+
+
+One last thing you might want to do before the snap is ready for wider consumption is to remove the `devmode` status.
+
+
+positive
+: Users of snaps using `devmode`, will need to pass `--devmode` during the installation, so they explicitly agree to trust you and your snap. Another benefit of removing devmode is that you will be able to ship your snap on the **stable** or **candidate** channels (you can only publish to the other channels, like **beta** or **edge** as your snap is less trusted) and users will be able to search for it using `snap find`.
+
```
Comment (degville) [inline tutorials/snapcraft/create-your-first-snap.md]: This is red/warning in the original:

image

File: tutorials/snapcraft/create-your-first-snap.md (hunk 1 above)
Comment (evilnick) [inline tutorials/snapcraft/create-your-first-snap.md]: oh, that is an actual mistake. The admonitions come through the conversion as code blocks and need to be restyled. I missed this one

File: tutorials/snapcraft/create-your-first-snap.md (hunk 2 above)
Comment (evilnick) [inline tutorials/snapcraft/create-your-first-snap.md]: ok, I just chose the wrong type here :) Will fix

File: tutorials/snapcraft/create-your-first-snap.md (hunk 1 above)
Comment (evilnick) [inline tutorials/snapcraft/create-your-first-snap.md]: done

File: tutorials/snapcraft/create-your-first-snap.md (hunk 2 above)
Comment (evilnick) [inline tutorials/snapcraft/create-your-first-snap.md]: done

**Comment by evilnick** [comment]
Thanks for the fullsome review @degville

**Pull request merged**